README.md

```diff
@@ -447,33 +445,51 @@ In some cases, the latest features or bug fixes may be integrated into the main
 
 <sub>**Warning:** _Features and bug fixes that are implemented on the latest version of Terrain may still be subject to testing. As such, you should only use the main branch of the Terrain github repo in exceptional circumstances. In all other cases, use the npm package._</sub>
 
-To use the main branch of the Terrain repo on your local machine, do the following:
+To use the main branch of the Terrain repo on your local machine, follow the procedure below.
 
-1. Clone the repo.
+1. Ensure the terrain package has been installed globally.
+
+```sh
+npm install -g @terra-money/terrain
+```
+
+2. Clone the repo.
 
 ```
 git clone --branch main --depth 1 https://github.com/terra-money/terrain
 ```
 
-2. Navigate to the project folder.
+3. Navigate to the project folder.
 
 ```
 cd terrain
 ```
 
-3. Inside the project folder, install all necessary node dependencies.
+4. Inside the project folder, install all necessary node dependencies.
 
 ```
 npm install
 ```
 
-4.  Run the `npm link` command to link the project to your global terrain instance.
```

Comment:
@octalmage On step 5 it says to run npm link to link the local terrain repo to your global terrain instance.  So I just thought it would make more sense to start with how to install terrain globally.